pnpm-lock.yaml

```diff
@@ -8085,6 +8085,7 @@ packages:
 
   /debug@4.1.1:
     resolution: {integrity: sha512-pYAIzeRo8J6KPEaJ0VWOh5Pzkbw/RetuzehGM7QRRX5he4fPHx2rdKMB256ehJCkX+XRQm16eZLqLNS8RSZXZw==}
+    deprecated: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
```

Comment:
@styfle why did `pnpm` add these deprecation messages to the lock file?

Reply:
Not sure. Could have changed since we swapped pnpm versions

Reply:
https://github.com/pnpm/pnpm/issues/5772#issuecomment-1344807194